utils/constants.py

```diff
@@ -0,0 +1,86 @@
+from web3 import Web3
+
```

Comment:
This and all the files below are from the official Uniswap V2 implementation. Would you happen to know if they are required for your use case? What changes have you guys made?

P.S. - Please describe the use case and how these compute modules fit in the README file.